specification/recoveryservicesbackup/resource-manager/readme.md

```diff
@@ -29,7 +29,6 @@ title: Recovery Services Backup Client
 description: Open API 2.0 Specs for Azure RecoveryServices Backup service
 openapi-type: arm
 tag: package-2017-07
-output-folder: ./
 csharp-sdks-folder: ./Generated/CSharp

```

Comment:
is this intentionally removed ?

Reply:
Yes, we specify output folders per language.